src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -153,6 +153,7 @@
     <Compile Include="$(PathToMsalSources)\Platforms\netcore\**\*.cs" LinkBase="Platforms\netcore" />
     <Compile Include="$(PathToMsalSources)\Platforms\Features\DefaultOSBrowser\**\*.cs" />
     <Compile Include="$(PathToMsalSources)\Platforms\Features\DesktopOs\**\*.cs" />
+    <Compile Include="$(PathToMsalSources)\Platforms\Features\WinFormsLegacyWebUi\**\*.cs" />
```

Comment:
this is not going to work because .net core 2.1 does not have win forms. this needs to be in Msal.Desktop DLL

Reply:
Yep. Removed.